security-connectors-api/src/main/java/fish/payara/security/annotations/OpenIdAuthenticationDefinition.java

```diff
@@ -359,4 +366,10 @@
      * The Microprofile Config key for evaluating EL expressions for every HTTP session is <code>{@value}</code>
      */
     String OPENID_MP_SESSION_SCOPED_CONFIGURATION = "payara.security.openid.sessionScopedConfiguration";
+
+    /**
+     * The Microprofile Config key to skip the /userinfo endpoint call
+     * and get the user information from ID Token is <code>{@value}</code>
+     */
+    String OPENID_MP_CLAIMS_FROM_ID_TOKEN = "payara.security.openid.claimsFromIDToken";
```

Comment:
I suggest we are consistent with rest of code and call the property `userClaimsFromIdToken` -- we are not capitalizing abbreviations in other properties (URI is the only exception), and `user` is important distinction.

Reply:
I already fix it, thanks